packages/provider/src/HocuspocusProvider.ts

```diff
@@ -96,6 +96,16 @@ export interface CompleteHocuspocusProviderConfiguration {
    * Don’t output any warnings.
    */
   quiet: boolean,
+
+  /**
+   * Pass `false` to start the connection manually.
+   */
+  connect: boolean,
+
+  /**
+   * Pass `false` to close the connection manually.
+   */
+  disconnect: boolean,
```

Comment:
Just thinking if we can find a better name, but so far I haven't been able to come up with something more self-explanatory (and also not as long as `propagateDisconnectToWebsocket`)

Definitely should be added to `docs/provider/configuration.md`

Reply:
Name ideas:

- `autodisconnect`
- `passthroughDisconnect`

Alternatively, if you inverted the value it could be called:

- `preserveConnection`
- `stayConnected`
- `keepaliveConnection`

Reply:
Thanks! `preserveConnection` sounds good 😎